### ISSUE TITLE
Simplify issue states

### ISSUE THREAD

**Issue by nijel** [phpmyadmin/error-reporting-server#120]
As for incident tracking, there should be less states, most of current ones are not really useful.

I think that following could be best fit:
- New
- Forwarded (set automatically once issue is linked with issue)
- Resolved (see also #98)

**Comment by devenbansod** [comment]
I think we should also migrate old ones to these new ones.
In the migration, first we would run query to update status of whichever issues have already been linked to any Github issue to 'forwarded'.

We can run a batch to get all the issues on issue tracker labelled 'automated-error-report) (currently only 58 (open+closed) in total) and set the status of reports which are linked to these issues according to the status of related issue. 

Once this is done, we could move to map the states of all the reports to:
* Duplicate: Check the status of one which it is duplicate of and set this to the same.
* Fixed: Resolved
* Invalid: Invalid
* New: New
* Open: New
* Out of date: Resolved
* Won't fix: Resolved
* Works for me: Resolved
